yarn.lock

```diff
@@ -2,36 +2,72 @@
 # yarn lockfile v1
```

Comment:
If you're removing `ember-welcome-page`, could you also update `yarn.lock`?

Reply:
Of course, don't know how I missed that.